exchanges/coinut/coinut_wrapper.go

```diff
@@ -551,10 +551,8 @@ func (c *COINUT) GetActiveOrders(getOrdersRequest *exchange.GetOrdersRequest) ([
 	for instrument, allInstrumentData := range instruments.Instruments {
 		for _, instrumentData := range allInstrumentData {
 			for _, currency := range getOrdersRequest.Currencies {
```

Comment:
In our tests we keep currency pairs slice as zero which should = all currencies enabled. This just skips over that process.

Reply:
I have added a todo check for all exchanges GetActiveOrder functions as a trello item.